static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -353,6 +361,40 @@ class GroupDetails extends Component<Props, State> {
     GroupStore.onPopulateReleases(this.props.params.groupId, releases);
   }
 
+  async fetchReplaysCount() {
+    const {api, location, organization, params} = this.props;
+    const {groupId} = params;
+    const {slug: orgId} = organization;
+
+    this.setState({loadingReplaysCount: true});
+
+    const eventView = EventView.fromSavedQuery({
+      id: '',
+      name: `Replays in issue ${groupId}`,
+      version: 2,
+      fields: ['count()'],
+      query: `issue.id:${groupId}`,
+      projects: [],
+    });
+
+    try {
+      const [data] = await doDiscoverQuery<TableData>(
+        api,
+        `/organizations/${orgId}/eventsv2/`,
```

Comment:
No need for this intermediate variable, also we can use `/events/` instead of `v2`

```suggestion
        `/organizations/${organization.slug}/events/`,
```